src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -889,6 +919,18 @@ private void modifyCollaborators(Collection<GHUser> users, String method) throws
         }
     }
 
+    private void modifyCollaborators(GHOrganization.Permission perm, Collection<GHUser> users, String method)
```

Comment:
Can we make all versions call one method?  Pass `null` for permission if it isn't used? 
```suggestion
    private void modifyCollaborators(@Nonnull Collection<GHUser> users, @Nonnull String method, @CheckForNull GHOrganization.Permission perm)
```